packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -1852,7 +1855,7 @@ function scrollAreaStateReducer(
 // Alternatively we could reconsider exposing the Position component, or we could allow users to
 // pass their own positionRef with a prop.
 function useExtendedScrollAreaRef(
-  forwardedRef: React.ForwardedRef<any>,
+  forwardedRef: React.Ref<any>,
```

Comment:
No idea what happened with this file! But somehow this TS error got flagged because `React.ForwardedRef` is not a thing. I have no idea how this went through for months…

On top of that all the other changes are just prettier, again not sure how it went in not formatted…

For both reasons above I've declined a package update for this as it's just internal.